Source/LocalNetAppChat/LocalNetAppChat.Bot/PluginProcessor/Plugins/PowerShellPlugin.cs

```diff
@@ -0,0 +1,62 @@
+﻿using LocalNetAppChat.Domain.Serverside;
+using LocalNetAppChat.Domain.Shared;
+using System.Management.Automation;
+
+namespace LocalNetAppChat.Bot.PluginProcessor.Plugins
+{
+    public class PowerShellPlugin : IPlugin
+    {
+        private readonly string _pluginsFolder = Directory.GetCurrentDirectory() + "..\\..\\my_scripts";
```

Comment:
Should be a parameter when it is not a draft anymore

Reply:
💯